contracts/cairo/utils/common.cairo

```diff
@@ -46,3 +55,53 @@ func u256_pow{range_check_ptr}(base: felt, exp: felt) -> (res: Uint256) {
         return (res,);
     }
 }
+
```

Comment:
maybe add here a comment on the exact purpose of the function?

Reply:
// Returns the number of digits needed to represent `num` in hexadecimal.